docs/spec/spec.md

```diff
@@ -641,16 +704,18 @@ The negatively-acknowledged (rejected) messages can be re-queued by setting the
    
    listener rabbitmq:Listener channelListener =
            new(rabbitmq:DEFAULT_HOST, rabbitmq:DEFAULT_PORT);
+           
+   public type Person record {|
+       string name;
+       int age;
+   |};  
    
    @rabbitmq:ServiceConfig {
        queueName: "MyQueue"
    }
    service rabbitmq:Service on channelListener {
-       remote function onRequest(rabbitmq:Message message) returns string {
-           string|error messageContent = string:fromBytes(message.content);
-           if messageContent is string {
-               log:printInfo("Received message: " + messageContent);
-           } 
+       remote function onRequest(Person person) returns string {
+           log:printInfo("Received person data: " + person.toString());
            return "Hello back from ballerina!";
```

Comment:
I think we can do `return Hello + person.name;`

Reply:
fixed in https://github.com/ballerina-platform/module-ballerinax-rabbitmq/pull/399/commits/3a0c50ad81de8fcbe2dab219b7971de22ca3be2e